src/processors/knex-processor.ts

```diff
@@ -1,11 +1,27 @@
 import * as Knex from "knex";
 import * as pluralize from "pluralize";
+import { camelize } from 'ember-cli-string-utils';
 
 import Resource from "../resource";
 import { KnexRecord, Operation, ResourceConstructor } from "../types";
 
 import OperationProcessor from "./operation-processor";
 
+const operators = {
+  eq: '=',
+  ne: '!=',
+  lt: '<',
+  gt: '>',
+  le: '<=',
+  ge: '>=',
+  like: 'like',
+  in: 'in',
+  nin: 'not in'
+};
+
+const getOperator = (paramValue: any) =>
+  operators[Object.keys(operators).find(operator => paramValue.indexOf(`${operator}:`) === 0)];
```

Comment:
@aklkv We can use `.startsWith()` instead of `.indexOf()` - feels more readable.